modules/util/path.go

```diff
@@ -283,38 +283,47 @@ func CommonSkip(name string) bool {
 	return false
 }
 
-// IsReadmeFileName reports whether name looks like a README file
+type FileType string
```

Comment:
I don't think both readme and license have the same logic. i.e. README files can be readme.zh.md, readme, .docs/readme, and much more.
So maybe we don't need the abstract layer for these two.

Reply:
Your examples can be a format: `path/filetype.lang.ext`
I think they have the same logic. 🤔

Reply:
Yep, I thought about it as well.
And my conclusion was that they indeed share the same search mechanism:
There are people calling it `license`, `LICENSE`, `license.md`, `LICENSE.md`, `license.txt`, …
And those are just the ones I've seen so far.
So yes, apart from the `.docs`/`.github` folders, they equal.

Reply:
... you forgot `COPYRIGHT`,  `COPYRIGHT.md`, ... :laughing: